components/measurement/nettests/Tor.js

```diff
@@ -235,7 +235,7 @@ const TorDetails = ({
   const messages = defineMessages({
     tor: {
       id: 'Measurement.Metadata.Tor',
-      defaultMessage: 'See results of Tor Test on {date} in {country} and other measurements on OONI Explorer.'
+      defaultMessage: 'Tor censorship test result in {country}'
```

Comment:
Why not use the `isAnomaly` flag?

Reply:
I just copied what already existed and I didn't want to make any changes. If we can easily refactor it to include different text for anomaly that would be great.